lib/mobility.rb

```diff
@@ -95,6 +95,14 @@ def extended(model_class)
       def model_class.translates(*args, **options)
         include Mobility.translations_class.new(*args, **options)
       end
+
+      # FIXME move this elsewhere?
+      def model_class.has_translated_rich_text(name)
+        has_rich_text(name)
+        has_one :"rich_text_#{name}", -> { where(name: name, locale: I18n.locale) },
+                class_name: "ActionText::RichText", as: :record, inverse_of: :record, autosave: true, dependent: :destroy
```

Comment:
You should be able to add this to a `setup` block called from within `Mobility::Backends::ActionText`, like this:

```ruby
module Mobility
  module Backends
    class ActionText < ActiveRecord::KeyValue
      setup do |attributes, _options|
        attributes.each do |name|
          has_rich_text(name) }
          has_one :"rich_text_#{name}", -> { where(name: name, locale: I18n.locale) },
            class_name: "ActionText::RichText", as: :record, inverse_of: :record, autosave: true, dependent: :destroy
        end
      end
    end
  end
end
```

By defining it this way, you now only need to call `translates` and `has_rich_text` will be called for you (so no need for `has_translated_rich_text`.)